contrib/babelfishpg_tsql/src/rolecmds.c

```diff
@@ -91,8 +94,22 @@ create_bbf_authid_login_ext(CreateRoleStmt *stmt)
 			if (defel->arg)
 				default_database = strVal(defel->arg);
 		}
+		else if (strcmp(defel->defname, "original_login_name") == 0)
+		{
+			if (defel->arg)
+			{
+				orig_loginname = strVal(defel->arg);
+			}
+		}
+		else if (strcmp(defel->defname, "from_windows") == 0)
+		{
+			from_windows = true;
+		}
 	}
 
+	if(!orig_loginname)
```

Comment:
When do we expect orig_loginname to be null? Otherwise, this should an Assert instead.

Reply:
when we call sys.initialize_babelfish('jdbc_user') to initialise the Babelfish.